foundry.toml

```diff
@@ -0,0 +1,12 @@
+[profile.default]
+src = 'contracts'
+out = "out"
+libs = ['node_modules','lib']
+test = 'foundry_test'
+solc = '0.8.25'
+
+[fmt]
+ignore = ['./contracts/**/*']
```

Comment:
@note here i'm opting to avoid linting the contracts all together. it can be something to consider tho once all issues/prs are close